### PR TITLE
Doc : Update CONTRIBUTING.md #827

### PR DESCRIPTION
**Pull request by outlivo**
Fix : issue https://github.com/dyrector-io/dyrectorio/issues/827

Updating CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Contributing
 
-We'd love you to contribute to [dyrector.io](https://dyrector.io/) platform. We want contribution to dyrector.io to be fun, enjoyable, and educational for anyone and everyone. All contributions are welcome, including issues, new docs as well as updates and tweaks, blog posts, workshops, and more.
+We would ❤️ you to contribute to [Dyrector.io](https://Dyrector.io/) and help make it better! We want contributing to Dyrector to be fun, enjoyable, and educational for anyone and everyone. All contributions are welcome, including issues, and new docs, as well as updates and tweaks, blog posts, workshops, and more.
```
Comment (polaroi8d) [inline CONTRIBUTING.md:3]: Please do not use emojis instead of letters.

Comment (polaroi8d) [reply CONTRIBUTING.md:3]: The correct way is: dyrector.io not Dyre...

**Review by polaroi8d — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 ## Code of Conduct
 
-Help us keep [dyrector.io](https://dyrector.io/) open and inclusive. Please read and follow our [Code of Conduct](/CODE_OF_CONDUCT.md).
+Help us keep Dyrector.io open and inclusive. Please read and follow our [Code of Conduct](/CODE_OF_CONDUCT.md).
 
 ## How to Start?
 
-If you're worried or don’t know where to start, check out our next section explaining what kind of help we could use and where you can become involved. You can reach out with questions to our [Discord server](https://discord.gg/hMyT9cbYFD) or [@dyrectorio](https://twitter.com/dyrectorio) on Twitter.
+If you're worried or don’t know where to start, check out our next section explaining what kind of help we could use and where you can become involved. You can reach us out with questions to [@dyrectorio](https://twitter.com/dyrectorio) on Twitter or to anyone from the [Dyrector.io team on Discord](https://discord.gg/hMyT9cbYFD) .
```
Comment (robot9706) [inline CONTRIBUTING.md:11]: I think `reach us out` is not correct. The original text should work just fine.

**Comment by robot9706** [comment]
Also please read the [CONTRIBUTING.md](https://github.com/dyrector-io/dyrectorio/blob/develop/CONTRIBUTING.md#submit-a-pull-request) to see how you can submit a pull request (branch name, PR title etc).

**Pull request closed without merging**